src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -374,6 +374,27 @@ public void MaxRequestContentBufferSize_Set_ThrowsPlatformNotSupportedException(
             }
         }
 
+        [Fact]
+        public async Task SendAsync_ReadFromSlowStreamingServer_PartialDataReturned()
+        {
+            // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
+            const string SlowStreamingServer = "http://httpbin.org/drip?numbytes=8192&duration=15&delay=1&code=200";
```

Comment:
It's not really slow. At most, if the test fails, it would take 15 seconds to drip the 8192 bytes. But when the test passes, the ReadAsync() call will return much faster with partial data.